app/assets/javascripts/client/views/assessments/permissions/manage_permission.html

```diff
@@ -60,7 +60,7 @@
             <p class="info">
               <fieldset>
                 <label class="level_label">Current Permission Level: </label>
-                <select class="permission-level" name="level">
+                <select class="permission-level" name="level" ng-disabled="user.role === 'network_partner'">
```

Comment:
So I take back what I said about the form not being disabled; it very much is.  The problem is that it doesn't _appear_ to be at first glance.